client/Elements/Components/TopicConsumer/TopicConsumerDetail.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import React from 'react';
+import {
+  DrawerPanelContent,
+  DrawerHead,
+  DrawerPanelBody,
+  DrawerActions,
+  DrawerCloseButton,
+  Flex,
+  FlexItem,
+  TextContent,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  TableVariant,
+} from '@patternfly/react-table';
+import { mockConsumerGroupDetail } from './ConsumerGroupDetail';
+import { ITopicConsumer } from './SearchTopicsConsumerList.patternfly';
+
+export interface ISearchTopicsConsumerListProps {
+  isExpanded: (value: boolean) => void;
+  consumerGroupDetail: ITopicConsumer[];
```

Comment:
Instead of the entire list of consumer groups, we should accept only the Consumer group whose detail has to be shown. So the type of consumerGroupDetail should be ITopicConsumer

Reply:
type consumergroupdetail is of ITopicConsumer which has the details of only the consumer group whose detail has to be shown. Is that what you meant?

Reply:
yes